src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -18,6 +18,8 @@ import { fetchAllManagingTeams } from '../../actions/team';
 import { getUserProfile } from '../../actions/userProfile';
 import Loading from '../common/Loading';
 import DiffedText from './DiffedText';
+import { getcolor } from '../../utils/effortColors'
```

Comment:
Nice reuse of the color part!

Reply:
Thanks for the review.

1. Are you looking at this on mobile view?
2. No, the color is based on the hours remaining on a task, not the amount of hours logged into a task based on the requirement above. I am not sure the end product image is correct here.
3. If the logged time is greater than the estimated, it should remain a red color based on requirement above.

Reply:
1. I looked this on my laptop. When I resized the window it looks like this. 
2. Yep I also saw this point and got focused, **maybe check with Jae.**
3. Understood! Great!

Reply:
The text in your first image is correct. The second posted image does not match the text. Colors for those bars should match how much time _remains_ for the task.

Reply:
The problem for #1 looks to be fixed now and new commit made. Please review again. 

Thanks